<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_Dialog</class>
 <widget class="QDialog" name="customer_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>6</y>
     <width>392</width>
     <height>270</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="name_comboBox">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="manufacturer_comboBox">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="category_comboBox">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>130</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="package_insert_comboBox">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  厂商：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  说明：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detail_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/7675352_090252138517_2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>230</y>
     <width>8</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>222</y>
     <width>221</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line_2">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>0</y>
     <width>8</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line_3">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>0</y>
     <width>8</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>0</y>
     <width>409</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>274</y>
     <width>409</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>272</y>
     <width>221</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>tableWidget</zorder>
  <zorder>name_comboBox</zorder>
  <zorder>manufacturer_comboBox</zorder>
  <zorder>category_comboBox</zorder>
  <zorder>package_insert_comboBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>query_pushButton</zorder>
  <zorder>detail_pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_line_2</zorder>
  <zorder>label_line_3</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
